lib/field_struct/avro_schema/kafka.rb

```diff
@@ -99,7 +99,11 @@ def register_event_schemas
       def register_event_schema(klass)
         return nil unless klass.publishable?
 
-        id = schema_registry.register build_subject_name(klass), klass.schema
+        id = if ENV.fetch('KAFKA_AUTO_REGISTER', 'false') == 'true'
```

Comment:
Please consider using the standard env var name for this: `KAFKA_AUTO_REGISTER_SCHEMAS`